README.md

```diff
@@ -17,3 +17,14 @@ page](https://docs.github.com/en/get-started/writing-on-github/working-with-adva
 might help with the notation for mathematical expressions.
 
 $f(n)\in o(g(n)) \iff \forall c>0, \exists n_0, \forall n\ge n_0: f(n) < c g(n)$
+
+## NOT STRICTLY FORMAL PROOF:
+If a function f(n) grows much slower than g(n) (i.e., f(n)∈o(g(n))), it also means that f(n)grows no faster than g(n) (i.e., f(n)∈O(g(n))). 
+
+This is because f(n)∈o(g(n)) implies that for any positive constant c, starting from some point n0, f(n) will always be less than c⋅g(n).
+
+The definition of O(g(n)): states that there exist constants c′ and n0′ such that f(n) is bounded above by c′⋅g(n) for all n≥n0′. 
+
+By choosing a c′ greater than any cc we choose for o(g(n)), and keeping n0′=n0, we ensure that f(n) is still bounded by c′⋅g(n) for all n≥n0.
```

Comment:
You can't choose a specific constant for $o$, you have to show that it holds true for all constants.